src/crafting_gui.cpp

```diff
@@ -241,8 +241,9 @@ const recipe *select_crafting_recipe( int &batch_size )
             auto all_items_filter = r->get_component_filter( recipe_filter_flags::none );
             auto no_rotten_filter = r->get_component_filter( recipe_filter_flags::no_rotten );
             const deduped_requirement_data &req = r->deduped_requirements();
+            has_proficiencies = r->character_has_required_proficiencies( get_player_character() );
```

Comment:
It looks like you're only considering proficiencies of the avatar, not any NPCs helping them.  Is that intentionally deferred to the future?

Reply:
It skipped my mind while implementing this, and I think that would be better to leave off until later. I can add it now it you feel it should be, however.